### PR TITLE
Basic Sentiment API

### PR DESCRIPTION
**Pull request by caraujo777**
Basic Sentiment API: Displays score next to each message. For old messages that don't have the score property, shows score 0.0

### CODE REVIEW

**Review by osjuga — APPROVED** [review]
Looks good!

**Review by pabloemilio97 — APPROVED** [review]

**Review by pabloemilio97 — APPROVED** [review]

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -41,6 +45,17 @@ public void init() {
     datastore = new Datastore();
   }
 
+  private float getSentimentScore(String text) throws IOException {
+  Document doc = Document.newBuilder()
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:49]: fix indent in this function

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ function showMessageFormIfLoggedIn() {
             aboutMeForm.classList.remove('hidden');
           }
         }
-      
+
```
Comment (gauravmishra) [inline src/main/webapp/js/user-page-loader.js:50]: remove unnecessary whitespace

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ function fetchAboutMe() {
   const url = '/about?user=' + parameterUsername;
   fetch(url)
       .then((response) => {
-        return response.text(); 
+        return response.text();
```
Comment (gauravmishra) [inline src/main/webapp/js/user-page-loader.js:81]: remove unnecessary whitespace

File: src/main/webapp/js/user-page-loader.js [hunk 4]
```diff
@@ -97,8 +97,9 @@ function buildMessageDiv(message) {
   const headerDiv = document.createElement('div');
   headerDiv.classList.add('message-header');
   headerDiv.appendChild(document.createTextNode(
-      message.user + ' - ' + new Date(message.timestamp)));
-
+      message.user + ' - ' +
```
Comment (gauravmishra) [inline src/main/webapp/js/user-page-loader.js:100]: suggestion: since this expression is quite long now, it could be a bit hard to understand when going through the code quickly. How about initialize the message first and then use it here, e.g.:

```
String message = message.user + ' - ' + new Date(message.timestamp) +
      ' Sentiment Score: [' + message.sentimentScore + ']';
headerDiv.appendChild(document.createTextNode(message));
```

**Review by gauravmishra — APPROVED** [review]
Great job!
